src/translations/hu.json

```diff
@@ -36,6 +36,7 @@
   "continue_with": "Bejelentkezés ezzel:",
   "No user found with this email": "Nem található fiók ezzel az e-mail címmel",
   "Invalid Password": "Helytelen Jelszó",
+  "No user found with that email, or password invalid.": "No user found with that email, or password invalid.",
```

Comment:
Nem található fiók ezzel az e-mail címmel, vagy helytelen jelszó.

Reply:
I was not sure about the "vagy" part, thank you very much 👍